test/functionalTest/cases/0876_entity_dates/entity_dates_overridden_by_user_subs.test

```diff
@@ -462,6 +462,11 @@ Fiware-Correlator: REGEX([0-9a-f\-]{36})
                 "type": "Text",
                 "value": "bar"
             },
+            "dateModified": {
```

Comment:
Old implemenation was buggy and dateModified were not included in the notification. After render changes, the bug has been solved, so this .test is changing.

Reply:
NTC (I guess)

Reply:
NTC